src/status_im/chat/styles/input/input.cljs

```diff
@@ -82,8 +82,8 @@
    :ios                 {:line-height min-input-height
                          :left        (+ 10 left)}})
 
-(defnstyle seq-input-text [left]
-  {:min-width           200
+(defnstyle seq-input-text [left container-width]
```

Comment:
I tried it with no `:min-width` specified and the result was effectively multiline-input, where overflowing text wraps and continues on the next line. 
The problem was that this (multiline-input) for some reason broke `/command` input parsing, maybe because of `/newline` characters present in the input, so I rather retained the single-line input.